driver-sync/src/test/functional/com/mongodb/client/AbstractExplainTest.java

```diff
@@ -99,27 +99,48 @@ public void testExplainOfAggregateWithNewResponseStructure() {
         AggregateIterable<BsonDocument> iterable = collection
                 .aggregate(singletonList(Aggregates.match(Filters.eq("_id", 1))));
 
-        Document explainDocument = iterable.explain();
-        assertNotNull(explainDocument);
+        Document explainDocument = getAggregateExplainDocument(iterable.explain());
```

Comment:
There's an explanatory comment related to such a structure change at the top of this method, should it be updated / removed?

Reply:
Good idea.  That's not quite the place for it (since it's about skipping the tests) but I added a note above the new helper method instead.